### ISSUE TITLE
Model Validation Analysis

### ISSUE THREAD

**Issue by nathiedeheyl** [nathiedeheyl/pp5_mildew-detection#22]
As I trained the model and plottet learning curves on train and validation image datasets, I noticed a deviation from what the example learning curves looked like. While the train accuracy curve turned out like expected, exponentially growing, the validation accuracy curve seemed remarkably flat and started fairly high rather than low near zero on validation accuracy levels, and vice versa for the loss levels, but no clear indication of overfitting.

I investigated this to ensure model reliability.

The model achieves consistent high accuracy on a separate test set that was not used during training or validation, which I think confirms genuine predictive capability.

This slightly atypical learning pattern can be attributed to the distinct visual differences between healthy and infected leaves, since I decided to not downsize the image on the dataset, so the model was able to capture differences effectively from early training epochs and pass back high results.
